### PR TITLE
Add some css and reset Button

### PR DESCRIPTION
**Pull request by HassanFdim**
I added some CSS and a rest Button to the Code

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const p1Button = document.querySelector("#p1button");
 const p2Button = document.querySelector("#p2button");
+const resetButton = document.querySelector("#reset");
 const p1Display = document.querySelector("#p1Display");
 const p2Display = document.querySelector("#p2Display");
+const winningScoreSelect = document.querySelector("#playto");
```
Comment (Abdallah01win) [inline app.js:6]:
```suggestion
const winningScoreSelect = document.querySelector("#playto");

```

File: app.js [hunk 2]
```diff
@@ -20,7 +24,23 @@ p2Button.addEventListener("click", function () {
     p2Score += 1;
     if (p2Score === winningScore) {
       isGameOver = true;
+      p2Display.classList.add("winner");
+      p1Display.classList.add("loser");
     }
     p2Display.textContent = p2Score;
   }
 });
+winningScoreSelect.addEventListener("change", function () {
```
Comment (Abdallah01win) [inline app.js:36]:
```suggestion

winningScoreSelect.addEventListener("change", function () {
```

File: index.html [hunk 3]
```diff
@@ -4,9 +4,28 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Score keeper</title>
+    <style>
+      .wnner {
```
Comment (Abdallah01win) [inline index.html]:
```suggestion
      .winner {
```

File: app.js [hunk 4]
```diff
@@ -20,7 +24,23 @@ p2Button.addEventListener("click", function () {
...
+  winningScore = parseInt(this.value);
+  reset();
+});
+resetButton.addEventListener("click", reset);
```
Comment (Abdallah01win) [inline app.js:41]:
```suggestion

resetButton.addEventListener("click", reset);
```

**Review by Abdallah01win — CHANGES_REQUESTED** [review]
Add spaces & fix 'winner' class type

**Review by HassanFdim — COMMENTED** [review]
i made the changes

**Review by Abdallah01win — CHANGES_REQUESTED** [review]
Spaces

**Review by HassanFdim — COMMENTED** [review]
i added missed spaces

**Review by Abdallah01win — APPROVED** [review]

**Pull request merged**